installer/modelsDLTK.py

```diff
@@ -36,12 +36,12 @@ def addModels(login: str, modelPath: str, update: bool):
 	logger(f"Creating the {tgzFileModel} model.",forceConsoleOutput=True)
 	runJob("tar czf %s %s" % (tgzFileModel, modelName), cwd=modelsDir)
 	
-	logger(yellow("Warning: Uploading, please BE CAREFUL! This can be dangerous."))
-	logger(f'You are going to be connected to {login} to write in folder {SCIPION_SOFTWARE_EM}.')
-	if input("Continue? YES/no\n").lower() == 'no':
+	logger(yellow("Warning: Uploading, please BE CAREFUL! This can be dangerous."),forceConsoleOutput=True)
+	logger(f'You are going to be connected to {login} to write in folder {SCIPION_SOFTWARE_EM}.',forceConsoleOutput=True)
+	if input("Continue? YES/no\n").lower() != 'YES':
```

Comment:
This will never work, because you are converting the input result into lowercase, so it can never be `YES`:
```
input("Continue? YES/no\n")
YeS --> YeS.lower() --> yes
yes != YES
```